package.json

```diff
@@ -43,7 +43,7 @@
 	},
 	"engines": {
 		"npm": ">=7.0.0",
-		"node": ">=14.0.0"
+		"node": ">=14.0.0 <=16.18.1"
```

Comment:
@kendallstrautman this will break if a new 16.x patch release is adopted 🤔

Reply:
cool, i'll use the .x syntax instead. I wasn't sure if that's what was messing with the bundle analysis check